lcdb/reporting.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python
+""" Reporting System """
+
+import os
+import sys
+import base64
+from textwrap import dedent
+
+import markdown
+import jinja2
+
+TEMPLATES_DIR = os.path.join(os.path.dirname(__file__), 'templates')
+loader = jinja2.FileSystemLoader(TEMPLATES_DIR)
+ENV = jinja2.Environment(loader=loader)
+
+def report(string, **kwargs):
+    """ Build a report """
+    md = markdown.Markdown(output_format='html5')
+    tp = ENV.get_template('base.html')
+    return tp.render(content=md.convert(dedent(string).format(**kwargs)))
+
```

Comment:
Still trying to decide how to build table and figure html representations. Right now I am doing it with a rule and saving the HTML chunk. Then importing the report rule. If we think this is best then probably should add a file importing here instead of assuming `**kwargs` is a string ready for injection.